src/components/editMeasure/details/MeasureMetaDataTable.scss

```diff
@@ -1,65 +0,0 @@
-$border-color: #8c8c8c;
-
```

Comment:
deleting this file might affect CSS for the reference table?

Reply:
@adongare added back.